src/main/java/com/develhope/spring/order/dto/OrderMapper.java

```diff
@@ -23,6 +26,20 @@ public class OrderMapper {
     @Autowired
     private VehicleRepository vehicleRepository;
 
+    public static OrderDTO toDTO(Order order) {
```

Comment:
se mi approvate il refactor generico di order possiamo eliminare questo mapper, tiriamo tutti un pull nei branch attivi e pushiamo (visto che i DTO ora sono diversi in creazione ed in risposta)